src/styles/blocks/calendar.scss

```
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto ;
+
+  gap: $gap;
+  padding: $padding;
+  max-width: 7*$date-width + 2*$padding + 6*$gap;
+
+  @for $index from 28 through 31 {
+    &--month-length--#{$index} {
+      & :nth-child(n + #{$index + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  @for $i from 1 through 31 {
+    &__date-#{$i} {
+      height: $date-height;
+      width: $date-width;
+      background-color: #eee;
+      border: 1px solid black;
+      cursor: pointer;
+    }
+
+    &__date-#{$i}::before {
+      content: "#{$i}";
+      font-size: 30px;
+      font-family: Arial, Helvetica, sans-serif;
+      color: black;
+    }
+
+    &__date-#{$i}:hover {
+      background-color: pink;
+    }
```

You could simple write 
```suggestion
    &__date {
    //here add styles for &__date
      &:hover {
        background-color: pink;
      }
    }
```